src/rust/ide/view/graph-editor/src/component/node/output/port.rs

```diff
@@ -34,8 +33,7 @@ const PORT_OPACITY_NOT_HOVERED : f32 = 0.25;
 const SEGMENT_GAP_WIDTH        : f32 = 2.0;
 const HOVER_AREA_PADDING       : f32 = 20.0;
 const INFINITE                 : f32 = 99999.0;
-
-const TOOLTIP_LOCATION : Placement = Placement::Bottom;
+const LABEL_OFFSET             : f32 = -30.0;
```

Comment:
What port labels have to do with types under the node? This change seems completely unrelated to the subject of this PR.

**EDIT:** Ah, these are not port labels. They were meant to be output type labels! The naming have to be changed eveywhere to reflect that.